pkg/machine/qemu/machine.go

```diff
@@ -605,10 +605,12 @@ func CheckActiveVM() (bool, string, error) {
 // startHostNetworking runs a binary on the host system that allows users
 // to setup port forwarding to the podman virtual machine
 func (v *MachineVM) startHostNetworking() error {
-	binary := filepath.Join("/usr/lib/podman/", machine.ForwarderBinaryName)
+	// TODO we may wish to configure the directory in containers common
+	binary := filepath.Join("/usr/libexec/podman/", machine.ForwarderBinaryName)
 	if _, err := os.Stat(binary); os.IsNotExist(err) {
-		return errors.Errorf("unable to find %s", binary)
+		return errors.Errorf("unable to find %s", filepath.Join("/usr/libexec/podman/", binary))
```

Comment:
Shouldn't this remain just `binary`?

Reply:
Yes this should be binary.

Reply:
Actually this is a stutter, just return err

Reply:
Why do we care if the err is NotExist versus some other err?